website_blog_no_background_image/README.rst

```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+No background images in website Blog
+================================
```

Comment:
Hi @gfcapalbo Thanks for the contribution
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Underline until end of sentence.